pure_protobuf/protobuf.py

```diff
@@ -353,134 +412,186 @@ def dump(self, fp, value):
                         UVarint.dump(fp, key)
                         field_type.dump(fp, single_value)
             elif self.__has_flag(tag, Flags.REQUIRED, Flags.REQUIRED_MASK):
-                raise ValueError('The field with the tag %s is required but a value is missing.' % tag)
-        
+                msg = (
+                    "The field with the tag {} is required, "
+                    "but a value is missing."
+                )
+                raise ValueError(msg.format(tag))
+
     def load(self, fp):
-        fp, message = EofWrapper(fp), self.__call__() # Wrap fp and create a new instance.
+        fp, message = (
+            EofWrapper(fp),
+            self.__call__(),
+        )  # Wrap fp and create a new instance.
         while True:
             try:
                 tag, wire_type = _unpack_key(UVarint.load(fp))
                 if tag in self.__tags_to_types:
                     field_type = self.__tags_to_types[tag]
-                    if not self.__has_flag(tag, Flags.PACKED_REPEATED, Flags.REPEATED_MASK):
+                    if not self.__has_flag(
+                        tag, Flags.PACKED_REPEATED, Flags.REPEATED_MASK
+                    ):
                         if wire_type != field_type.WIRE_TYPE:
-                            raise TypeError(
-                                'The received value with the tag %s has incorrect wiretype: %s instead of %s expected.' % \
-                                (tag, wire_type, field_type.WIRE_TYPE))
+                            msg = (
+                                "The received value with the tag {} "
+                                "has incorrect wiretype: {} "
+                                "instead of {} expected."
+                            ).format(tag, wire_type, field_type.WIRE_TYPE)
+                            raise TypeError(msg)
                     elif wire_type != Bytes.WIRE_TYPE:
-                        raise TypeError('Tag %s has wiretype %s while the field is packed repeated.' % (tag, wire_type))
+                        msg = (
+                            "Tag %s has wiretype {} while the field is "
+                            "packed repeated."
+                        ).format(tag, wire_type)
+                        raise TypeError(msg)
                     if self.__has_flag(tag, Flags.SINGLE, Flags.REPEATED_MASK):
                         # Single value.
-                        message[self.__tags_to_names[tag]] = field_type.load(fp)
-                    elif self.__has_flag(tag, Flags.PACKED_REPEATED, Flags.REPEATED_MASK):
+                        message[self.__tags_to_names[tag]] = field_type.load(
+                            fp
```

Comment:
Mmm, moving 3 chars to separate lines looks a bit weird. Though I'm not strictly against it :)

Reply:
I'll blame this on `black`, I can re-do any weirdness it introduced. I figured that was an easier way to get `flake8` compliance.